### PR TITLE
Add support for GIT submodules

### PR DESCRIPTION
**Pull request by aledpardo**
This PR is an enhancement to #21.

### CODE REVIEW

**Comment by aledpardo** [comment]
Alright, @samcontesse I made some updates to the way the GIT modules are initialized.

Kindly ask your review.

**Review by samcontesse — COMMENTED** [review]
Thank you for this improvement. Did you have the chance to test it ?

File: in/cmd/main.go [hunk 1]
```diff
@@ -54,6 +55,16 @@ func main() {
 	execGitCommand([]string{"remote", "update"})
 	execGitCommand([]string{"merge", "--no-ff", "--no-commit", mr.SHA})
 
+	// After merging, clone submodules if git submodules file is present
+	if request.Source.Submodules != "none" && fileExists(".gitmodules") {
+		var sourceGitlabHost = request.Source.GetGitlabServerDomain()
+
+		var targetGitlabHost = "gitlab-ci-token:" + request.Source.PrivateToken + "@" + sourceGitlabHost
+
+		execSedCommand([]string{"-i", "s/" + sourceGitlabHost + "/" + targetGitlabHost + "/g", ".gitmodules"})
```
[the remark this answers is not in the document]
Comment (aledpardo) [reply in/cmd/main.go:64]: Sure! I'll give it a try @samcontesse.

**Comment by samcontesse** [comment]
Closing due to inactivity. Support for submodules has been added in https://github.com/samcontesse/gitlab-merge-request-resource/pull/89.

**Pull request closed without merging**